lib/cocoapods/target/aggregate_target.rb

```diff
@@ -7,6 +7,9 @@ class AggregateTarget < Target
     #         generated this target.
     attr_reader :target_definition
 
+    # Product types where the product's frameworks must be embedded in a host target
+    EMBED_FRAMEWORKS_IN_HOST_TARGET_TYPES = [:app_extension, :watch_extension].freeze
```

Comment:
This is still a bit confusing to me, but I think some clarity is coming. Are we talking about a situation wherein CocoaPods is integrating a framework target that is then supposed to be embedded in an app? In this case, do we expect this dependency to be communicated in Xcode manually via a target's dependency list or in the Podfile via target inheritance?